frontend/src/main/webapp/pagevisit.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Place Visit</title>
+     <script src="pagevisit.js"></script>
+     <style>
```

Comment:
I'm not absolutely sure, but I don't think we can have in line style or java.

Reply:
yeah I didn't plan on keeping it it just I didn't have Css file